lib/chttp/http.go

```diff
@@ -47,8 +47,8 @@ func (t *transport) RoundTrip(req *http.Request) (*http.Response, error) {
 	return resp, nil
 }
 
-func NewTransport(name string, t http.RoundTripper) *transport {
-	return &transport{name, t}
+func NewTransport(name string, t http.RoundTripper) *Transport {
```

Comment:
lib/chttp/http.go:50:53: unexported-return: exported func NewTransport returns unexported type *chttp.transport, which can be annoying to use (revive)
```
func NewTransport(name string, t http.RoundTripper) *transport {
                                                    ^
```